Source/WebKit/UIProcess/mac/WindowServerConnection.mm

```diff
@@ -26,13 +26,14 @@
 #import "config.h"
 #import "WindowServerConnection.h"
 
-#if PLATFORM(MAC)
+#if PLATFORM(MAC) || PLATFORM(MACCATALYST)
 
 #import "WebProcessPool.h"
 #import <pal/spi/cg/CoreGraphicsSPI.h>
 
 namespace WebKit {
 
+#if PLATFORM(MAC)
```

Comment:
Should this be Catalyst? I worry this won't compile without allowing it, but maybe it's not called anywhere in a Catalyst flow?